core/src/main/java/org/keycloak/representations/docker/DockerError.java

```diff
@@ -60,7 +60,7 @@ public boolean equals(final Object o) {
 
         final DockerError that = (DockerError) o;
 
-        if (errorCode != that.errorCode) return false;
+        if (!errorCode.equals(that.errorCode)) return false;
```

Comment:
Consider a null safe operation here:

```suggestion
        if (!Objects.equals(errorCode, that.errorCode)) return false;
```